src/Games/NexusMods.Games.MountAndBlade2Bannerlord/Diagnostics/Diagnostics.cs

```diff
@@ -55,7 +55,7 @@ 2. Enable mod `{DependencyId}`
     internal static IDiagnosticTemplate MissingVersionRangeTemplate = DiagnosticTemplateBuilder
         .Start()
         .WithId(new DiagnosticId(Source, 1))
-        .WithTitle("'{ModName}' is Missing Dependency with ID '{DependencyId}', version '{VersionRange}'")
+        .WithTitle("Missing Version Range")
```

Comment:
Title should be `Missing Dependency`. `Missing Version Range` makes no sense at all.